app/models/ext_management_system.rb

```diff
@@ -108,7 +108,8 @@ def hostname_uniqueness_valid?
            :port=,
            :security_protocol,
            :security_protocol=,
-           :to => :default_endpoint
+           :to => :default_endpoint,
+           :allow_nil => true
```

Comment:
ugh. is this only for what I changed, or is this a problem with other code as well?